pkg/apis/cluster/v1alpha1/types.go

```diff
@@ -215,14 +309,33 @@ type ResourceSummary struct {
 	// Total amount of allocatable resources on all nodes.
 	// +optional
 	Allocatable corev1.ResourceList `json:"allocatable,omitempty"`
+
 	// Allocating represents the resources of a cluster that are pending for scheduling.
 	// Total amount of required resources of all Pods that are waiting for scheduling.
 	// +optional
 	Allocating corev1.ResourceList `json:"allocating,omitempty"`
+
 	// Allocated represents the resources of a cluster that have been scheduled.
 	// Total amount of required resources of all Pods that have been scheduled to nodes.
 	// +optional
 	Allocated corev1.ResourceList `json:"allocated,omitempty"`
+
+	// AllocatableModeling represents the number of each resource modeling in a cluster that are available for scheduling.
+	// Total amount of allocatable resources on all nodes.
+	// +optional
+	AllocatableModeling []AllocatableModelingItem `json:"allocatableModeling,omitempty"`
+}
+
+// AllocatableModelingItem represents the number of this resources modeling in a cluster that are available for scheduling.
+// For example, AllocatableModelingItem{Grade: 2, Count: 10}, which means there are 10 such resource models of grade 2 ResourceModel.
+type AllocatableModelingItem struct {
```

Comment:
```suggestion
+// AllocatableModeling represents the number of nodes in which allocatable resources in a specific resource model grade.
+// E.g. AllocatableModeling{Grade: 2, Count: 10} means 10 nodes belong to resource model in grade 2.
+type AllocatableModeling struct {
```

Rename `AllocatableModelingItem` --> `AllocatableModeling`?

Reply:
Done.